### PR TITLE
[Feature] Support classmethods in tensorclass

### PR DESCRIPTION
**Pull request by vmoens**

### CODE REVIEW

File: tensordict/tensorclass.py [hunk 1]
```diff
@@ -443,6 +437,37 @@ def wrapped_func(*args, **kwargs):
     return wrapped_func
 
 
+def _wrap_method(cls, func):
+    @functools.wraps(func)
+    def wrapped_func(*args, **kwargs):
+        print(cls, args, func.__get__(cls))
```
Comment (tcbegley) [inline tensordict/tensorclass.py]: Was this just for debugging?

**Review by apbard — APPROVED** [review]
LGTM! thanks! left couple of comments

File: tensordict/tensorclass.py [hunk 2]
```diff
@@ -406,21 +414,7 @@ def wrapper(self, key: str, value: Any) -> None:  # noqa: D417
...
-    Returns:
-        value of the attribute, or a method output applied on the instance
-
-    """
-    res = getattr(self._tensordict, attr)
-    if not callable(res):
-        return res
-    func = res
-
+def _wrap_func(self, attr, func):
```
Comment (tcbegley) [inline tensordict/tensorclass.py:417]: Perhaps we could call this `_wrap_method` and rename `_wrap_method` to `_wrap_classmethod`?

**Review by tcbegley — APPROVED** [review]
LGTM, one minor comment on naming which I found a bit confusing

**Pull request merged**